### ISSUE TITLE
No delivery day in OrderItem wrong params and tag naming

### ISSUE THREAD

**Issue by bartdigitalpulse** [distrimedia/soap-client#1]
In DistriMedia\SoapClient\Struct\OrderItem the NO_DELIVERY_DAY_<dayname> constants should be called NoDelivery_<day> instead of NoDelivery_Day<day>.

The properties $noDelivery<day> are nog longer booleans but chars with the value 'F' or 'T' so the getNoDelivery<day> and setNoDelivery<day> methods should accept a string as a param. 

Thanks for this lib, it was very helpful!

**Comment by duckchip** [comment]
Hi @bartdigitalpulse, thx for your comment. 

I think it's better if we output the NO_DELIVERY_DAYS to T or F in the xml, but keep the use of the booleans. It makes more sense from a logical perspective to use booleans. 

So in the end the xml will contain T or F but internally we use booleans.

Also, you can set each individual day using the getters and setters. Why would you want to change the constants? 

Thx

**Comment by bartdigitalpulse** [comment]
Will there be a situation where the constants will have a different value? It seems like extra work to use setters.

**Comment by duckchip** [comment]
@bartdigitalpulse can you try to install dev-v1-develop  version of this module? 
This includes https://github.com/distrimedia/soap-client/pull/2 

The setters have to be used, I cannot change that.

**Comment by duckchip** [comment]
@bartdigitalpulse have you tried this out yet? :)

**Comment by bartdigitalpulse** [comment]
Yes this work for the boolean values. Thanks for the fix!

I still have to add a extension to change these values 

from:
```
    const NO_DELIVERY_DAY_MONDAY = 'NoDelivery_DayMonday';
    const NO_DELIVERY_DAY_TUESDAY = 'NoDelivery_DayTuesday';
    const NO_DELIVERY_DAY_WEDNESDAY = 'NoDelivery_DayWednesday';
    const NO_DELIVERY_DAY_THURSDAY = 'NoDelivery_DayThursday';
    const NO_DELIVERY_DAY_FRIDAY = 'NoDelivery_DayFriday';
    const NO_DELIVERY_DAY_SATURDAY = 'NoDelivery_DaySaturday';
    const NO_DELIVERY_DAY_SUNDAY = 'NoDelivery_DaySunday';
```

to:
```
  const NO_DELIVERY_DAY_MONDAY = 'NoDelivery_Monday';
  const NO_DELIVERY_DAY_TUESDAY = 'NoDelivery_Tuesday';
  const NO_DELIVERY_DAY_WEDNESDAY = 'NoDelivery_Wednesday';
  const NO_DELIVERY_DAY_THURSDAY = 'NoDelivery_Thursday';
  const NO_DELIVERY_DAY_FRIDAY = 'NoDelivery_Friday';
  const NO_DELIVERY_DAY_SATURDAY = 'NoDelivery_Saturday';
  const NO_DELIVERY_DAY_SUNDAY = 'NoDelivery_Sunday';
```

**Comment by duckchip** [comment]
Hi @bartdigitalpulse , I will change those constants too.

**Comment by duckchip** [comment]
This issue is fixed in 1.0.2
